docs/apps/interchain-accounts/auth-modules.md

```diff
@@ -1,9 +1,13 @@
 <!--
-order: 2
+order: 6
 -->
 
 # Building an authentication module
 
+### Deprecation Notice
```

Comment:
Can we add a new section before `RegisterInterchainAccounts` and `SendTx` which is labeled `Legacy API`?

Reply:
Sure, I can add one now. Just something you can link to in your use cases doc?

Is a level 3 heading okay?

Reply:
yup thanks!